components/icons/shopping-cart.tsx

```diff
@@ -0,0 +1,41 @@
+export default function ShoppingCartIcon({ className }: { className?: string }) {
+  return (
+    <svg
+      className={className}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Comment:
Can we clean this up a bit?

```
<svg
   xmlns="http://www.w3.org/2000/svg"
   viewBox="0 0 16 16"
   fill="none"
   stroke="currentColor"
   strokeWidth="1.5"
   strokeLinecap="round"
   strokeLinejoin="round"
   shapeRendering="geometricPrecision"
   className={className}
 >
```

Then remove `stroke`, `stroke-width`, `stroke-linecap`, `stroke-linejoin`, and `fill` from below.